app/routes/member/index.tsx

```diff
@@ -0,0 +1,81 @@
+import type { ActionArgs, LoaderArgs } from "@remix-run/node";
+import { useLoaderData, Link, Form } from "@remix-run/react";
+import { FaMobileAlt } from "react-icons/fa";
+import { requireMemberId, getMemberId } from "~/memberSession.server";
+import { json, redirect } from "@remix-run/node";
+import { getUpcomingInvitations, getMemberById } from "~/models/member.server";
+import parseISO from "date-fns/parseISO";
+import intlFormat from "date-fns/intlFormat";
+import { useUser } from "~/utils";
+import { InvitationListItem } from "~/components/invitation_list_item";
+
+export async function loader({ request }: LoaderArgs) {
+  await requireMemberId(request);
+
+  const memberId = await getMemberId(request);
+
+  if (!memberId) throw redirect("/");
```

Comment:
If we do that, this line won't be necessary because `requireMemberId` _always_ returns an id, making the null check here redundant